src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final int DATA_TYPE_INDEX = 0;
+    private static final int DATA_VALUE_INDEX = 1;
+    private static final int SUM_SUPPLY_INDEX = 0;
+    private static final int SUM_BUY_INDEX = 1;
+    private static final int SUM_RESULT_INDEX = 2;
+    private int[] sum = new int[3];
+    private final StringBuilder builder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            File file = new File(fromFileName);
+            readFile(file);
+            file = new File(toFileName);
+            writeFile(file);
+
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot create file", e);
+        }
```

Comment:
Seems like this message relates to the method `writeFile `. Let's move try-catch block to methods where you the error can occur